pkg/kubectl/cmd/get.go

```diff
@@ -218,6 +219,20 @@ func RunGet(f *cmdutil.Factory, out io.Writer, cmd *cobra.Command, args []string
 		Do()
 	err = r.Err()
 	if err != nil {
+		// providing an invalid cluster or context flag will yield
+		// an invalid configuration error when the client config
+		// fails to be obtained. If such error is received and either
+		// flag is set, such flag's value can then be blamed
+		if clientcmd.IsConfigurationMissing(err) {
+			context := cmdutil.GetFlagString(cmd, "context")
+			cluster := cmdutil.GetFlagString(cmd, "cluster")
```

Comment:
We should definitely not be looking up specific flags by name and erroring here. If we error, it should be when we attempt to apply the command line override flags at point of use